doc/operator-notes.md

```diff
@@ -30,3 +30,35 @@ In the second two cases, where there is a preexisting filesystem, Ignition's beh
 If `wipeFilesystem` is set to true, Ignition will always wipe any preexisting filesystem and create the desired filesystem. Note this will result in any data on the old filesystem being lost.
 
 If `wipeFilesystem` is set to false, Ignition will then attempt to reuse the existing filesystem. If the filesystem is of the correct type, has a matching label, and has a matching UUID, then Ignition will reuse the filesystem. If the label or UUID is not set in the Ignition config, they don't need to match for Ignition to reuse the filesystem. Any preexisting data will be left on the device and will be available to the installation. If the preexisting filesystem is *not* of the correct type, then Ignition will fail, and the machine will fail to boot.
+
+## selinux
+
+When using Ignition with distributions which rely on [selinux][selinux] for additional security extra care must be taken to prevent Ignition from breaking the system. Unfortunately a typical setup in a distribution is to not include selinux policies in the initramfs where Ignition runs, so any files, directories, and links created by Ignition don't receive the proper default selinux labels they should.
+
+A workaround for this issue is to use [`restorecon`][restorecon] to relabel files that Ignition touched in a oneshot systemd unit. This unit can be set to run after the selinux policies have loaded, but before services will try to use them.
```

Comment:
`to use restorecon in a oneshot systemd unit to relabel files that Ignition has touched`.